frontend/js/helpers/Transaction.js

```diff
@@ -0,0 +1,131 @@
+//common functions that can be used in different cases
+
+
+async function Get(path) {
+    return await ApiRequest("http://localhost:3000/" + path, "GET");
+}
+
+async function Post(path, body) {
+    return await ApiRequest("http://localhost:3000/" + path, "POST", body);
+}
+
+async function ApiRequest(url, type, body) {
+    return await $.ajax({
+        url: url,
+        type: type,
+        data: JSON.stringify(body),
+        contentType: "application/json",
+        success: (data) => {
+            return data;
+        },
+        error: (err) => {
+            console.log(err);
+        },
+    });
+}
+
+
+//2023.8.25 
```

Comment:
Please delete this comment.
By looking git history, we can check when this codes are created.